benchmark/ts/typescript.js

```diff
@@ -0,0 +1,54 @@
+'use strict';
+
+const common = require('../common');
+const path = require('path');
+const assert = require('node:assert');
+
+
+// Strip types benchmark
+{
+  const js = path.resolve(__dirname, '../fixtures/strip-types-benchmark.js');
+  const ts = path.resolve(__dirname, '../fixtures/strip-types-benchmark.ts');
+
+  const bench = common.createBenchmark(main, {
+    filepath: [ts, js],
+    n: [1e4],
+  }, {
+    flags: ['--experimental-strip-types', '--disable-warning=ExperimentalWarning'],
+  });
+
+  async function main({ n, filepath }) {
+    let output;
+    bench.start();
+    for (let i = 0; i < n; i++) {
+      const { result } = await import(filepath);
+      output = result;
+    }
+    bench.end(n);
+    assert.ok(output);
+  }
+}
+
+// Transform types benchmark
+{
+  const js = path.resolve(__dirname, '../fixtures/transform-types-benchmark.js');
```

Comment:
Hmm... That's not what I meant. You should include the flags as a benchmark option. For instance:

```
  const bench = common.createBenchmark(main, {
    filepath: [ts, js],
    transformType: [0, 1],
    n: [1e4],
  }, {
    flags: ['--experimental-transform-types', '--disable-warning=ExperimentalWarning'],
  });
```

Reply:
Im not sure how to achieve this since flags are not configurable

Reply:
Doesn't `--experimental-transform-types` already enable `--experimental-strip-types`? If so, you don't need the former for this benchmark. If they are completely different branches, then forget what I said.

Reply:
`--experimental-transform-types` then source maps are enable and the comparison doest make sense, because its not strip types anymore